slf4cats-api/src/main/scala/slf4cats/api/slf4cats-api.scala

```diff
@@ -3,35 +3,24 @@ package slf4cats.api
 import cats.effect.Sync
 import org.slf4j.Marker
 
-trait LoggingContext[F[_]] {
+trait StructuredLoggerArguments[F[_]] {
```

Comment:
the point of this trait is to add things to context and then execute an action with the augmented context -- I'm not sure `StructuredLoggerArguments` communicates that well :man_shrugging:

Reply:
Actually its not renamed. Its incorrect diff. See bellow. I just disliked that `Logger` and `LoggingContext` had exact same methods withArgs and withComputed so I pulled them into trait which both inherit. I admit the name is not ideal.